app/src/main/res/values/strings.xml

```diff
@@ -610,5 +610,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="ask_to_turn_location_on">Turn on location?</string>
   <string name="nearby_needs_location">Nearby needs location enabled to work properly</string>
   <string name="use_location_from_similar_image">Did you shoot these two pictures at the same place? Do you want to use the latitude/longitude of the picture on the right?</string>
-
+  <string name="mapbox_telemetry">Telemetry Opt Out</string>
+  <string name="telemetry_opt_out_summary">Send anonymized location and usage data to Mapbox</string>
```

Comment:
Perhaps it might help make this more palatable for users if we add `when using "Nearby" feature` at the end of this sentence. Sorry for the multiple reviews @ashishkumar468 , I just thought about this.

Reply:
Done @misaochan